.github/workflows/cln-plugin.yaml

```diff
@@ -64,6 +64,7 @@ jobs:
           cd lightning && sudo make install
       - name: Install teos and the plugin
         run: |
+          sudo apt-get update && sudo apt-get install -y protobuf-compiler
```

Comment:
fyi, an old/outdated cache could be deleted from here: https://github.com/talaia-labs/rust-teos/actions/caches

Reply:
Trying that manually now to see if I can avoid that the last commit

Reply:
Ok, looks like the issue was not with the cache, but with loading the package. We are only caching the `lightning` folder, so protoc is not available after loading the cache. I'll try to change it to use an actions module for it, or just add the protoc installation unconditionally

Reply:
Ahaa.
We could cache the directory that protoc is installed in as well, like:
```yaml
path: |
    lightning
    /usr/bin/protoc
```

Reply:
Seems like adding the `protoc` GH action worked, testing it now also with the CLN installation

Reply:
It worked! :)